contracts/contract_state_tree.md

```diff
@@ -13,20 +14,51 @@ The contract state tree contains objects:
 - The contract log (data binary)
 - Active flag (Boolean)
 - List of refering contracts
-- Calls & Return data
 - Deposit (amount)
 - vm_version
 
+It is serialized as:
+```
+      CONTRACT_TYPE,
+      CONTRACT_VSN,
+      pubkey,
+      balance,
+      height,
+      nonce,
```

Comment:
I suggest deleting this paragraph from here and rather review https://github.com/aeternity/protocol/blob/b9b9eb2a707a18b7b976be78f3eca76e27c9c5e3/serializations.md#contract e.g. unclear (missing?) nonce there.

Reply:
Hmm we need to describe both how a transaction is serizlized and how a state tree is serialized. Do we descibe both somewhere in protocol now?

Reply:
Aha, we do in that version.